### PR TITLE
Vagrant provisioning rebased

### PR DESCRIPTION
**Pull request by jgrund**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -157,68 +157,105 @@ clean_substs:
...
 destroy_cluster: Vagrantfile
-	vagrant destroy -f
-	sed -ie '/# VAGRANT START/,/# VAGRANT END/d' ~/.ssh/config
-	sed -ie '/IML Vagrant cluster/d' ~/.ssh/authorized_keys
-	export LIBVIRT_DEFAULT_URI=qemu:///system;                       \
-	for net in intel-manager-for-lustre{0,1,2,3} vagrant-libvirt; do \
-	    virsh net-destroy $$net || true;                             \
-	    virsh net-undefine $$net || true;                            \
-	done
+	time vagrant destroy -f
```
Comment (johnsonw) [inline Makefile]: Is this mostly formatting?

File: Vagrantfile [hunk 2]
```diff
@@ -1,73 +1,108 @@
...
+
+if ENV['JENKINS'] == 'true'
+  ci_cluster = true
+  fqdn = Socket.gethostname
+  dot = fqdn.index('.')
+  hostname = fqdn[0..dot - 1]
+  domain = fqdn[dot + 1..-1]
+  mach_num = hostname[hostname.index("-") + 1..-1]
+  cluster_num = 0
+  if cluster_num == 0
```
Comment (johnsonw) [inline Vagrantfile]: Isn't cluster_num always 0?

File: Vagrantfile [hunk 3]
```diff
@@ -1,73 +1,108 @@
...
+    cluster_num_s = ""
+  else
+    cluster_num_s = cluster_num
+  end
+else
+    ci_cluster = false
+    hostname = ""
+    cluster_num_s = ""
```
Comment (johnsonw) [inline Vagrantfile]: if cluster_num is always 0 in the first if, you could just move cluster_num_s outside of the if statement.

File: Vagrantfile [hunk 4]
```diff
@@ -1,73 +1,108 @@
...
-	# Subnet index used to create cross-over nets for each HA cluster pair
-	xnet_idx=230
-
-	# Create a basic hosts file for the VMs.
-	open('hosts', 'w') { |f|
-	f.puts <<-__EOF
+    # Use box from Manager For Lustre project - this differs from the
+    # "official" CentOS base box by using a SATA controller instead of IDE.
+    # This simplifies the addition of extra disks for the storage servers.
+    config.vm.box = "manager-for-lustre/centos73-1611-base"
```
Comment (johnsonw) [inline Vagrantfile]: Why centos73 and not the latest?

File: tests/framework/utils/cluster_setup.sh [hunk 5]
```diff
@@ -5,7 +5,7 @@ mkdir -p $PWD/test_reports
 mkdir -p $PWD/coverage_reports
 
 CLUSTER_CONFIG=${CLUSTER_CONFIG:-"$(ls $PWD/shared_storage_configuration_cluster_cfg.json)"}
-CHROMA_DIR=${CHROMA_DIR:-"$PWD/integrated-manager-for-lustre/"}
+CHROMA_DIR=${CHROMA_DIR:-"$PWD/intel-manager-for-lustre/"}
```
Comment (johnsonw) [inline tests/framework/utils/cluster_setup.sh]: why "intel"?

Comment (jgrund) [reply tests/framework/utils/cluster_setup.sh]: fixing.

**Pull request merged**